packages/frontend2/src/app/(new)/(other)/scaling/liveness/_components/table/columns.tsx

```diff
@@ -0,0 +1,155 @@
+import { type ScalingLivenessTableEntry } from './to-table-entry'
+
+const columnHelper = createColumnHelper<ScalingLivenessTableEntry>()
+
+export const columns = [
+  // TODO: Make sure this is centered
+  columnHelper.accessor((_, index) => index + 1, {
+    header: '#',
+    cell: (ctx) => <IndexCell>{ctx.row.index + 1}</IndexCell>,
+    meta: {
+      headClassName: 'w-0',
+      cellClassName: 'flex items-center',
+    },
+  }),
+  columnHelper.display({
+    id: 'logo',
+    cell: (ctx) => (
+      <Image
+        className="min-h-[18px] min-w-[18px]"
+        src={`/icons/${ctx.row.original.slug}.png`}
+        width={18}
+        height={18}
+        alt={`${ctx.row.original.name} logo`}
+      />
+    ),
+    meta: {
+      headClassName: 'w-0',
+      cellClassName: '!pr-0 md:pl-1',
+    },
+  }),
+  columnHelper.accessor('name', {
+    cell: (ctx) => (
+      <ProjectNameCell
+        project={ctx.row.original}
+        type={ctx.row.original.type}
+      />
+    ),
+  }),
+  columnHelper.group({
+    id: 'data',
+    header: () => <IntervalsHeader average={true} />,
+    columns: [
+      columnHelper.accessor('data.batchSubmissions.averageInSeconds', {
+        header: 'Tx data\nsubmissions',
+        cell: (ctx) => (
+          <LivenessIntervalCell
+            entry={ctx.row.original}
+            dataType="batchSubmissions"
+          />
+        ),
+        sortUndefined: 'last',
+        meta: {
+          tooltip: 'How often transaction batches are submitted to the L1',
+        },
+      }),
+      columnHelper.accessor('data.proofSubmissions.averageInSeconds', {
+        header: 'Proof\nsubmissions',
+        cell: (ctx) => (
+          <LivenessIntervalCell
+            entry={ctx.row.original}
+            dataType="proofSubmissions"
+          />
+        ),
+        sortUndefined: 'last',
+        meta: {
+          tooltip: 'How often validity proofs are submitted to the L1',
+        },
+      }),
+      columnHelper.accessor('data.stateUpdates.averageInSeconds', {
+        header: 'State\nupdates',
+        cell: (ctx) => (
+          <LivenessIntervalCell
+            entry={ctx.row.original}
+            dataType="stateUpdates"
+          />
+        ),
+        sortUndefined: 'last',
+        meta: {
+          tooltip: 'How often state roots are submitted to the L1',
+        },
+      }),
+    ],
+  }),
+  columnHelper.accessor('category', {
+    header: 'Type',
+    cell: (ctx) => (
+      <TypeCell provider={ctx.row.original.provider}>{ctx.getValue()}</TypeCell>
+    ),
+    meta: {
+      tooltip: <TypeExplanationTooltip showOnlyRollupsDefinitions />,
+    },
+  }),
+  columnHelper.accessor('anomalies', {
+    header: '30-day\nanomalies',
+    enableSorting: false,
+    cell: (ctx) => {
+      const entry = ctx.row.original
+      const showComingSoon =
+        !entry.data?.syncStatus.isSynced ||
+        entry.slug === 'linea' ||
+        entry.slug === 'starknet' ||
+        entry.slug === 'scroll'
```

Comment:
What's the deal with these? I thought lack of data would cause it to become 'coming soon'

Reply:
well, if i'm correct we have wrong data for anomalies of these projects so its manually set to coming soon, i will export it as variable somewhere